src/components/Modals/EditPoliticianManifestoModal.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <our-modal @close-modal="closeModal()" :open="true" size="large">
```

Comment:
Should this "true" be replaced by a variable?

Reply:
Weirdly enough that's the way it's meant to be, I made some changes along the way that made that necessary. I'll refactor how the modal works in the nearest future.

Reply:
Alright sounds good.